appveyor.yml

```diff
@@ -11,13 +11,19 @@ before_build:
 
 platform: Any CPU
 
+environment:
+  _CL_: /c /analyze /analyze:quiet- /W4 /analyze:autolog:ext prefast.sarif
+
 configuration:
 #  - Debug
   - Release
 
 build:
   project: src/Everything.sln
-  
+
+before_test:
```

Comment:
(And Ryley's point is correct -- beyond the question of code clarity, there really is a semantic difference.)